api-gateway/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM java:8
+ADD api-gateway.jar app.jar
```

Comment:
Is not really mandatory. Just by using `filtering` we can have generic `Dockerfile`